OpenRA.Mods.Common/Traits/Render/WithIdleOverlay.cs

```diff
@@ -67,7 +70,11 @@ public WithIdleOverlay(Actor self, WithIdleOverlayInfo info)
 
 			buildComplete = !self.HasTrait<Building>(); // always render instantly for units
 			overlay = new Animation(self.World, rs.GetImage(self));
-			overlay.PlayRepeating(RenderSprites.NormalizeSequence(overlay, self.GetDamageState(), info.Sequence));
+			if (Info.StartSequence != null)
```

Comment:
`base(info)` sets `Info = info`, so it does the same thing. I could change it, but is it worth it. I doubt it would make an impact on performance to get from a field instead of an argument.